example/thirdparty/ollama-js/build.mill

```diff
@@ -0,0 +1,224 @@
+package build
+
+import mill._, javascriptlib._
+import os.*
+
+trait OllamaModule extends TypeScriptModule {
+
+  // clean git source - remove generated json files
+  def cleanUp = Task {
+    os.checker.withValue(os.Checker.Nop) {
+      os.remove(T.workspace / "package.json")
+      os.remove(T.workspace / "package-lock.json")
+      os.remove(T.workspace / "tsconfig.json")
+    }
+  }
+
+  def compile = Task {
+    cleanUp()
+    super.compile()
+  }
```

Comment:
This `cleanUp` task is kind of weird. Why do we need to remove these global files every time we compile things? Can we instead have the `TypeScriptModule.compile` task ignore these files during compilation?

Reply:
Fixed - Mill will use local package.json & tsconfig.json if one is provided